wiki/ticket-splitting-v1.6.md

```diff
@@ -0,0 +1,145 @@
+
+## What is the short status?
+
+> We absolutely agree ticket splitting is worthwhile. Supporting it correctly moving forward is a lot of work that touches the entire stack, which is going to take some time. See this [blog post](https://blog.decred.org/2019/11/11/LN-Multi-Owner-Tickets/). Meanwhile you can still split tickets using the legacy tools, see [here](https://decredvoting.com/ticketsplitting).
+
+Thanks to u/Inaccurate- for [writing](https://www.reddit.com/r/decred/comments/m547to/not_focusing_on_split_tickets_will_affect_us/gr1iruu/) it so well.
+
+## Does it still work?
+
+Yes, as of writing (March 2021) buying partial tickets is still possible.
+
+## How do I split tickets now?
+
+It is currently possible if the following conditions are met:
+
+- Decrediton (GUI) or dcrwallet (command-line) must be of v1.5 series
+- the VSP must be of "legacy" type
+- the VSP must support ticket splitting
+
+In other words, it will _not_ work with v1.6 software, and it will _not_ work with VSPs of the new "vspd" type.
+
+You can get the latest v1.5 software [here](https://github.com/decred/decred-binaries/releases/tag/v1.5.2), which is v1.5.2 (released Aug 2020).
+
+And then you need a compatible VSP. Three are known to support ticket splitting:
+
+- [99split.com](https://99split.com/)
+- [decredvoting.com](https://decredvoting.com/)
+- [stake.decredbrasil.com](https://stake.decredbrasil.com/)
+
+Finally, you also need other people to buy a full ticket as a group. Of the three VSPs above, only decredvoting.com is having _active sessions_ recently. It also has extra features like detailed [notifications](https://www.reddit.com/r/decred/comments/9x4f9a/decredvotingcom_now_sending_splitticket_purchase/) and a monitoring [dashboard](https://www.reddit.com/r/decred/comments/hbofz4/tired_of_keeping_track_of_your_staking_activities/).
+
+You can see splitting sessions for all VSPs on [this page](https://mainnet-split-tickets.matheusd.com/) (links are outdated but the monitoring should work), or [this one](https://macelai.github.io/decred-split-stakepools/) by another developer.
+
+This document covers the status of ticket splitting in v1.6 but it's not a tutorial - see [this page](https://decredvoting.com/ticketsplitting) for a good entry point with all instructions linked, and then you can connect with other users in [@dcrtktsplit](https://t.me/dcrtktsplit) Telegram or [#ticketsplitting](https://chat.decred.org/#/room/#ticketsplitting:decred.org) on Matrix.
+
+Note: Here we only talk about _VSP staking_, because users who can [solo](https://docs.decred.org/advanced/solo-proof-of-stake-voting/) stake likely don't need ticket splitting.
+
+## Can I downgrade from v1.6?
+
+No. If you have upgraded to v1.6, the only way to use ticket splitting again is to install v1.5, re-download the blockchain and re-create the wallet from seed.
+
+{ can multiple versions of Decrediton be installed on Windows and macOS via our recommended install paths? }
```

Comment:
Not easily for lay users. All config files (including wallet db) are (OS-)user global, so only a single install of decrediton can run for a given user.

You could probably do this if you have multiple OS users and use a different version on each, but I haven't personally tested this config.